debian/po/pt_BR.po

```diff
@@ -377,109 +389,113 @@ msgstr ""
 msgid "Installing {packages}"
 msgstr "Instalando {packages}"
 
-#: ../../uaclient/messages/__init__.py:126
+#: ../../uaclient/messages/__init__.py:127
 #, python-brace-format
 msgid "Installing {title} packages"
 msgstr "Instalando pacotes do {title}"
 
-#: ../../uaclient/messages/__init__.py:127
+#: ../../uaclient/messages/__init__.py:128
 msgid "Installing required snaps"
 msgstr "Instalando snaps necessários"
 
-#: ../../uaclient/messages/__init__.py:129
+#: ../../uaclient/messages/__init__.py:130
 #, python-brace-format
 msgid "Installing required snap: {snap}"
 msgstr "Instalando snap necessário: {snap}"
 
-#: ../../uaclient/messages/__init__.py:132
+#: ../../uaclient/messages/__init__.py:133
 #, python-brace-format
 msgid "Skipping installing packages: {packages}"
 msgstr "Não instalando os pacotes: {packages}"
 
-#: ../../uaclient/messages/__init__.py:134
+#: ../../uaclient/messages/__init__.py:135
 #, python-brace-format
 msgid "Uninstalling {packages}"
 msgstr "Desinstalando {packages}"
 
-#: ../../uaclient/messages/__init__.py:136
+#: ../../uaclient/messages/__init__.py:137
 #, python-brace-format
 msgid "Failure when uninstalling {packages}"
 msgstr "Falha ao desinstalar {packages}"
 
-#: ../../uaclient/messages/__init__.py:139
+#: ../../uaclient/messages/__init__.py:140
 #, python-brace-format
 msgid "Cannot install package {package} version {version}"
 msgstr "Não foi possível instalar o pacote {package} versão {version}"
 
-#: ../../uaclient/messages/__init__.py:142
+#: ../../uaclient/messages/__init__.py:143
 msgid "Failure checking APT policy."
 msgstr "Falha ao verificar 'APT policy'"
 
-#: ../../uaclient/messages/__init__.py:143
+#: ../../uaclient/messages/__init__.py:144
 msgid "Updating package lists"
 msgstr "Atualizando lista de pacotes"
 
-#: ../../uaclient/messages/__init__.py:144
+#: ../../uaclient/messages/__init__.py:145
 #, python-brace-format
 msgid "Updating {name} package lists"
 msgstr "Atualizando lista de pacotes: {name}"
 
-#: ../../uaclient/messages/__init__.py:145
+#: ../../uaclient/messages/__init__.py:146
 msgid "APT update failed."
 msgstr "APT update falhou"
 
-#: ../../uaclient/messages/__init__.py:146
+#: ../../uaclient/messages/__init__.py:147
 msgid "APT install failed."
 msgstr "APT install falhou"
 
-#: ../../uaclient/messages/__init__.py:148
+#: ../../uaclient/messages/__init__.py:149
 #, python-brace-format
 msgid "Backing up {original} as {backup}"
 msgstr "Salvando {original} como {backup}"
 
-#: ../../uaclient/messages/__init__.py:149
+#: ../../uaclient/messages/__init__.py:150
 msgid "The following package(s) will be REMOVED:"
 msgstr ""
 
-#: ../../uaclient/messages/__init__.py:151
+#: ../../uaclient/messages/__init__.py:152
 msgid "The following package(s) will be reinstalled from the archive:"
 msgstr "O(s) pacote(s) à seguir serão reinstalados:"
 
-#: ../../uaclient/messages/__init__.py:162
-#, python-brace-format
+#: ../../uaclient/messages/__init__.py:163
+#, fuzzy, python-brace-format
```

Comment:
Since we know pt_BR ;), we should review all these fuzzy translations. There aren't many.